integration_tests/fixtures/notes.yaml

```diff
@@ -0,0 +1,291 @@
+---
+version: 1
+interactions:
+- request:
+    body: '{"name":"Test Customer Notes"}'
+    form: {}
+    headers:
+      Content-Type:
+      - application/json
+      User-Agent:
+      - chartmogul-go/v3
+    url: https://api.chartmogul.com/v1/data_sources
+    method: POST
+  response:
+    body: '{"uuid":"ds_9dad66ec-99cd-11ee-a3b1-272c7e7811c1","name":"Test Customer
+      Notes","system":"Import API","created_at":"2023-12-13T15:38:08.767Z","status":"idle"}'
+    headers:
+      Access-Control-Allow-Credentials:
+      - "true"
+      Connection:
+      - keep-alive
+      Content-Length:
+      - "157"
+      Content-Type:
+      - application/json
+      Date:
+      - Wed, 13 Dec 2023 15:38:08 GMT
+      Retry-Count:
+      - "0"
+    status: 201 Created
+    code: 201
+    duration: ""
+- request:
+    body: '{"data_source_uuid":"ds_9dad66ec-99cd-11ee-a3b1-272c7e7811c1","email":"soeun+staff@chartmogul.com","external_id":"ext_customer_1","name":"Test
+      Customer Notes"}'
+    form: {}
+    headers:
+      Content-Type:
+      - application/json
+      User-Agent:
+      - chartmogul-go/v3
+    url: https://api.chartmogul.com/v1/customers
+    method: POST
+  response:
+    body: '{"id":183831708,"uuid":"cus_9dc5aaf4-99cd-11ee-a3b2-4bab72a136d9","external_id":"ext_customer_1","name":"Test
+      Customer Notes","email":"soeun+staff@chartmogul.com","status":"New Lead","customer-since":null,"attributes":{"custom":{},"clearbit":{},"stripe":{},"tags":[]},"data_source_uuid":"ds_9dad66ec-99cd-11ee-a3b1-272c7e7811c1","data_source_uuids":["ds_9dad66ec-99cd-11ee-a3b1-272c7e7811c1"],"external_ids":["ext_customer_1"],"company":"","country":null,"state":null,"city":"","zip":null,"lead_created_at":null,"free_trial_started_at":null,"address":{"country":null,"state":null,"city":"","address_zip":null},"mrr":0,"arr":0,"billing-system-url":null,"chartmogul-url":"https://app.chartmogul.com/#/customers/183831708-Test_Customer_Notes","billing-system-type":"Import
```

Comment:
Maybe you don't want to have this email address on the public repo

Reply:
We already have few email addresses like [this line](https://github.com/chartmogul/chartmogul-go/blob/v4/integration_tests/fixtures/subscriptions.yaml#L83). But I'm curious if we have a test account for integration testing. I test with localhost with changing our api address [this line](https://github.com/chartmogul/chartmogul-go/blob/v4/chartmogul.go#L44)

But I got errors like this.
```
=== RUN   TestConnectSubscriptions
    connect_subscription_test.go:34: Request error: Post "http://localhost:3000/api/v1/data_sources": Requested interaction not found
```